### ISSUE TITLE
Typing Errors

### ISSUE THREAD

**Issue by ID6109** [keras-team/keras-io#1400]
I've noticed several typos throughout the codebase. I'll be sending PRs specific to sections of the code instead of one big PR so that it's more manageable. Thanks.

**Comment by sachinprasadhs** [comment]
Hi, Thanks for opening the issue.

I see you have already created PR, our team will review it, once it is merged, this issue will be closed automatically.

Going forward, you can directly create a PR instead of creating a separate issue. Thanks!
